src/routes/components/radio/+page.svelte

```diff
@@ -0,0 +1,130 @@
+<script lang="ts">
+	import radio from '$lib/components/ui/radio-button/radio.svelte?raw'
+	import radioGroup from '$lib/components/ui/radio-button/radio-group.svelte?raw'
+	import Code from '$lib/components/custom/code.svelte'
+	import TabBar from '$lib/components/custom/tab-bar/tab-bar.svelte'
+	import TabContent from '$lib/components/custom/tab-bar/tab-content.svelte'
+	import { onMount } from 'svelte'
+	import Select from '$lib/components/ui/select/select.svelte'
+	import Option from '$lib/components/ui/select/option.svelte'
+	import ComponentTemplate from '$lib/components/custom/component-template.svelte'
+	import Radio from '$lib/components/ui/radio-button/radio.svelte'
+	import RadioGroup from '$lib/components/ui/radio-button/radio-group.svelte'
+
+	type Dimension = 'default' | 'large' | 'compact' | 'small'
+	type Layout = 'vertical' | 'horizontal'
+
+	let radioCss: string = $state('Loading...')
+	let radioGroupCss: string = $state('Loading...')
+
+	let dimension: Dimension = $state('default')
+	let layout: Layout = $state('vertical')
+
+	// Svelte compiler breaks when it finds closing script tag, hence the need to make the template literal to have two parts
+	let useCode = $derived(
+		`<script lang="ts">
+    import RadioGroup from '$lib/components/ui/radio-button/radio-group.svelte'
+    import Radio from '$lib/components/ui/radio-button/radio.svelte'
+</script` +
+			`>
+
+<RadioGroup dimesion="${dimension}" layout="${layout}" name="rb">
+    {#snippet helperText()}
```

Comment:
It would be good to add a control to make the optional helper text turned on or off. If it is turned on, it would be good to have an input so that the text can be changed.